### DISCUSSION TITLE
SIgning up for GIthub was a terrible experience

### DISCUSSION THREAD

**Discussion by Gametrek** [community/community#21106] [Projects and Issues]
I do not have any love for Google at all for a variety of reasons and is the same reason why I stop using google. I should not have to use chrome browser just to sign up for Github at all. It is a huge security risk for my computer. Google is a poor-persons system and we need to move away from it.

I tried the following browsers
Win 10 IE
Win 11 IE
Ubuntu 64 FireFox ( HTML5 )
Basilisk / Palemoon ( my primary browser that actually works )
Seamonkey 
FireFox ( HTML5 )  even without any anti-scripts
Brave Browser ... which did work but the buttons did not light up and would not allow me to continue registering. I even lowered sheilds, 

..........

It was only until I loaded up Windows 11 Chrome Google that the buttons did work and I was able to see the capture. I am not a fan of using Windows 11 at all, to do any work at all unless somebody gives me an absolute reason.

..........

I am not a fan of google and this is the browser I had to use just to load your system. Beyond that I have no problem obtaining code, or using the website on a user basis.

..........

Github should not be colorful, playful, or anything polite looking. It should be boring like an Excel spreadsheet or how users feel about Libre office. Your using too much resources for a website like this. 

I took up some programming classes with code-lab ( which was simple looking and I could log in on various browsers ) and that is how simple this website should be.

**Comment by MrSarno** [comment]
I have never needed to use Chrome for GitHub.

I ran through the signup process using Firefox without any issues whatsoever;

<img width="1860" alt="Screenshot 2022-07-23 at 16 35 19" src="https://user-images.githubusercontent.com/32073555/180611943-c54cd18b-2db3-4921-bc5c-f45968691afc.png">

While Internet Explorer is unlikely to still be supported by GitHub & I would very strongly recommend that you cease using it - particularly in cases where you would be inputting sensitive information, such as account credentials - Firefox should work just fine.

My only conclusion, based on not being able to reproduce the issue, is either a browser setting or plugin that is interfering with GitHub's ability to use JavaScript (which may or may not be from the same domain - e.g. github.com loading scripts from githubassets.com).

If GitHub had a genuine incompatibility issue with Firefox, I wouldn't have been able to create the above screenshot.

I would assume the same is true for Pale Moon, though I admittedly don't use the browser myself. I've no idea about SeaMonkey, sorry.

**Comment by danielj23** [comment]
Gotta love when they present you with an unanswerable question.   

![2024-03-29 13_43_52-Join GitHub · GitHub — Mozilla Firefox](https://github.com/community/community/assets/725527/dab60dd6-e05c-44ac-b1c9-483bcb5befcf)
